DEPS

```diff
@@ -454,6 +454,9 @@ deps = {
   'src/third_party/dart/third_party/pkg/test_reflective_loader':
    Var('dart_git') + '/test_reflective_loader.git@c4c2d5c3f94a96f3fc79e9e28944fba391bc544c',
 
+  'src/third_party/dart/third_party/pkg/tools':
+   Var('dart_git') + '/tools.git@28b7be83589d0d7a6d44cbe6166b6673f08018fb',
```

Comment:
Does this need to be rolled forward to match the version in the Dart SDK DEPS file when the Dart SDK rolls into the engine?

Reply:
The https://github.com/flutter/buildroot/blob/master/tools/dart/create_updated_flutter_deps.py script recognizes entries in this format and will copy updates from Dart's DEPS

Reply:
Yes, adding a new dependency is a one-time operation, after that, it updates when the revision in Dart DEPS changes, just like the other deps.